contracts/Strategy.sol

```diff
@@ -501,8 +582,13 @@ contract Strategy is BaseStrategy {
         internal
         view
         override
-        returns (address[] memory)
-    {}
+        returns (address[] memory ret)
+    {
+        ret = new address[](3);
+        ret[0] = yVault.token();
+        ret[1] = wantAsVault.token();
+        ret[2] = address(yVault);
+    }
```

Comment:
@newmickymousse there were some discussions about the value of `protectedTokens` and consensus was that it is OK to leave it empty. Initial cause of its existence was to prevent ychad from rugging funds but in particular for debt-based strategies we want to have several ways to salvage potentially locked stuff. Let's remove it and save some gas!

Reply:
Sure! I didn't know, but makes sense